docs/api/data-views/create.asciidoc

```diff
@@ -105,3 +179,76 @@ The API returns the data view object:
     "data_view": {...}
 }
 --------------------------------------------------
+
+
+[[data-views-api-properties]]
+
+==== Properties of the `data_view` object:
+
+`title`::
+(Optional, string) Comma-separated list of data streams, indices, and aliases to search. Supports wildcards
+(`*`).
+
+`name`::
+(Optional, string) Human readable name to differentiate the data view.
+
+`id`::
+(Optional, string) Saved object ID.
+
+`type`::
+(Optional, string) When set to `rollup`, this is used to identify rollup data views.
+
+`typeMeta`::
+(Optional, object) When using rollup indices, this contains the field list for the rollup-specific
+data view API endpoints.
++
+.Properties of the typeMeta objects:
+[%collapsible%open]
+=====
+`aggs`:::
+(Required, object) A map of rollup of the aggregation restrictions by aggregation type and field name.
+
+`params`:::
+(Required, object) Properties for retrieving rollup fields
+=====
+
+`timeFieldName`::
+(Optional, string) Timestamp field name, for time-based data views.
+
+`sourceFilters`::
+(Optional, string[]) Array of field names to filter out in Kibana.
+
+`fieldAttrs`::
+(Optional, object) Map of field attributes by field name.
++
+.Properties of the fieldAttrs[fieldName] objects:
+[%collapsible%open]
+=====
+`customLabel`:::
+(Optional, string) Custom label for the field.
+
+`count`:::
+(Optional, number) Popularity count for the field.
+=====
+
+`runtimeFieldMap`::
+(Optional, object) Map of runtime field definitions by field name.
++
+.Properties of the runtimeFieldMap[fieldName] objects:
+[%collapsible%open]
+=====
+`type`:::
+(Required, string) Mapping type of the runtime field. See {ref}/mapping-types.html[Field data types].
+
+`script.source`:::
+(Required, string) Script of the runtime field.
+=====
+
+`fieldFormats`::
+(Optional, object) Map of field formats by field name.
+
+`allowNoIndex`::
+(Optional, boolean) Allows the data view to exist before data is present.
```

Comment:
Is this true by default? It seems I can create data views of `logstash-*` without having that data in my cluster.